app/views/exchanges/hbx_profiles/_invoice.html.slim

```diff
@@ -4,3 +4,4 @@ h1.heading-text Employers
 
 javascript:
   initializeDataTables()
+  disableSelectric = true
```

Comment:
Removing Selectric because of performance. This disable is specific to the ER Datatable.